packages/cli/lib/commands/upgrade.ts

```diff
@@ -0,0 +1,74 @@
+import { GoogleAnalytics, ProjectConfig, Util } from "@igniteui/cli-core";
+import { TemplateManager } from "../TemplateManager";
+
+let command: {
+	[name: string]: any,
+	templateManager: TemplateManager,
+	execute: (argv: any) => Promise<void>,
+	upgrade: (argv: any) => Promise<void>
+};
+
+// tslint:disable:object-literal-sort-keys
+command = {
+	command: "upgrade",
+	desc: "Upgrades Ignite UI Packages",
+	templateManager: null,
+	builder: {
+		"skip-install": {
+			alias: "s",
+			default: false,
+			describe: "Runs upgrade command without calling `npm install`",
+			type: "boolean"
+		}
+	},
+	async execute(argv) {
+
+		GoogleAnalytics.post({
+			t: "screenview",
+			cd: "Upgrade"
+		});
+
+		command.upgrade(argv);
+	},
+	async upgrade(argv) {
+		const config = ProjectConfig.getConfig();
+		const framework = config.project.framework;
+		const projectType = config.project.projectType;
+
+		switch (framework.toLowerCase()) {
+			case "jquery":
+				Util.log("Upgrading packages for jQuery projects is currently not supported!");
```

Comment:
There can be some sort of implementation calling `PackageManager.ensureIgniteUISource` and then `EmptyTemplate.upgradeIgniteUIPackages`. The implementation could in a separate PR though, just wanted to mention in.

Reply:
No - adding the new command to cli.ts was forgotten.

Reply:
@ViktorSlavov Agreed, though it will take a bit of refactoring. Will log a separate issue